mantidproject/mslice
# Combine Workspace Button

This adds a button to the GUI which runs the `MergeMD` algorithm on a set of (projected) workspaces. The main use case is for the HYSPEC instrument where the detector can move in scattering angle, and two scans are need to cover the full range of scattering angles. 

Further uses could be to combine equivalent datasets, but where no instrument or sample parameters changed but the user prefered to make a series of shorter runs rather than a longer one, in case some equipment fails and corrupts a run; or for memory reasons for event data.

**To test:**

Compile and run. Check that a `Combine` button now exists. Load the data from this [zip file](https://www.dropbox.com/s/yjaextzz9ndrglp/hyspec_test_nxspe.zip?dl=0) and check that the output looks like the picture in the issue #155 . 

Fixes #155.
